app/views/partials/_sidebar_dashboard.html.erb

```diff
@@ -9,6 +9,7 @@
     <% end %>
     <% if current_user.tasker? %>
       <%= link_to raw("<li class='side-nav-button'>Map Search</li>"), map_search_path %><hr>
+      <%= link_to raw("<li class='side-nav-button'>Taskee Skillsets</li>"), "/taskees/index" %><hr>
```

Comment:
@andela-mpitan There's a `My Skillsets` link on the side nav. Which displays all `current user skillsets`. Also handled in the `TaskeeSkillsetsController`.